types/Button.ts

```diff
@@ -1,5 +1,7 @@
 export interface ButtonProps {
   children: string;
+  btnType?: "button" | "submit" | "reset" | undefined;
```

Comment:
옵셔널 (`?`) 로 지정했다면 undefined 일 수 있다는 의미이므로 undefined 를 작성하지 않아도 됩니다 

```suggestion
  btnType?: "button" | "submit" | "reset";
```